src/cryptoadvance/specter/templates/wallet/components/wallet_pdf.jinja

```diff
@@ -242,8 +242,14 @@
                 xpubLines = "{{ key.xpub }}".split(/(.{36})/g);
                 xpubLines = xpubLines.filter(line => line !== '');
                 currentHeight += 6
+                doc.setLineDash([2, 2], 0);
+                doc.line(181,0,181,214)
+                doc.line(0,214,181,214)
```

Comment:
You're drawing those lines for every key in the wallet. Please move it out of the loop!